src/main/resources/data/alaskanativecraft/recipes/labrador_tea.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "minecraft:crafting_shapeless",
+  "group": "labrador_tea",
+  "ingredients": [
+    {
+      "item": "minecraft:glass_bottle"
+    },
+    {
+      "item": "minecraft:water_bucket"
+    },
+    {
+      "item": "alaskanativecraft:labrador_tea_leaves"
+    },
+    {
+      "item": "alaskanativecraft:labrador_tea_leaves"
+    },
+    {
+      "item": "alaskanativecraft:labrador_tea_leaves"
+    }
+  ],
+  "result": {
+    "item": "alaskanativecraft:labrador_tea"
+  }
+}
```

Comment:
This should be brewed in a brewing stand instead IMO.

Reply:
Not possible without mixin

Reply:
Never shy away from a mixin, they are wonderful!